migrations/frontend/1668184279_remove_git_status_column_from_gitserver_repos_table/up.sql

```diff
@@ -0,0 +1,14 @@
+-- Perform migration here.
+--
+-- See /migrations/README.md. Highlights:
+--  * Make migrations idempotent (use IF EXISTS)
+--  * Make migrations backwards-compatible (old readers/writers must continue to work)
+--  * If you are using CREATE INDEX CONCURRENTLY, then make sure that only one statement
+--    is defined per file, and that each such statement is NOT wrapped in a transaction.
+--    Each such migration must also declare "createIndexConcurrently: true" in their
+--    associated metadata.yaml file.
+--  * If you are modifying Postgres extensions, you must also declare "privileged: true"
+--    in the associated metadata.yaml file.
+
+ALTER TABLE gitserver_repos
+DROP COLUMN repo_status;
```

Comment:
All migrations have to be idempotent so that they can run multiple times if necessary.
For your migration here that means:

```suggestion
ALTER TABLE gitserver_repos DROP COLUMN IF EXISTS repo_status;
```

If it now runs the second time, it would just see the column is already gone and all will be fine.